client/src/lib/parse/data/PersonAndGroupParser.ts

```diff
@@ -259,7 +259,7 @@ export default class PersonAndGroupParser extends EntityParser {
             if (eq.id === config.aat.gender) {
               label = 'Gender'
             } else if (eq.id === config.aat.occupation) {
```

Comment:
@kamerynB there is another AAT for Role, which the frontend doesn't have right now. (It is http://vocab.getty.edu/aat/300435108). The ML update added this AAT. I think maybe the frontend do the same? Like check if `eq.id === config.aat.occupation || eq.id === config.aat.role`?

Reply:
This should be fixed now.